.github/CHANGELOG.md

```diff
@@ -71,4 +72,3 @@
 1. [ND] Add DME distances, VOR/ADF needles and functioning ADF2 - @blitzcaster (bltzcstr)
 1. [OVHD] Fixed Battery Indicator Colour - @nathaninnes (Nathan Innes)
 1. [MISC] Removed Fuel Patch from MSFS Update 1.8.3 - @nathaninnes (Nathan Innes)
-
```

Comment:
This commit changes the end of the file, probably inadvertantly. Editor setting for trailing newline?

Reply:
Removed although I don't think it was a problem ... It probably depends on my editor, yes ...

Reply:
Yeah, it's probably not an issue, it's just that in a repository as large as this if people casually commit these changes in completely unrelated PRs it may lead to unnecessary merge conflicts in future.

Reply:
Yes, more than fair!